Makefile

```diff
@@ -21,9 +24,12 @@ JAVAC_RELEASE_VERSION=7
 
 SOURCES := $(wildcard src/*.cpp)
 HEADERS := $(wildcard src/*.h src/fdtransfer/*.h)
-JAVA_HEADERS := $(patsubst %.java,%.class.h,$(wildcard src/helper/one/profiler/*.java))
+JAVA_HELPER_SOURCES := $(wildcard src/helper/one/profiler/*.java)
+JAVA_HELPER_CLASSES := $(patsubst src/helper/%.java,$(CPP_RES_DIR)/%.class,$(JAVA_HELPER_SOURCES))
```

Comment:
I'd put helper .class files next to corresponding .java sources
and .html resources - just in the flat `res` directory.

Reply:
done